CONTRIBUTING.md

```diff
@@ -91,13 +97,13 @@ $ npm run lint-fix
 Check formatting with [prettier](https://prettier.io/):
 
 ```bash
-$ npm run prettier
```

Comment:
I think this was meant to be the `format-check` command, based on the line above.

Reply:
Ooh I understand now, I swapped them. Let me fix! Thanks!